newIDE/app/src/BehaviorsEditor/index.js

```diff
@@ -155,6 +159,8 @@ const BehaviorsEditor = (props: Props) => {
                           onClick={ev => {
                             ev.stopPropagation();
                             onRemoveBehavior(behaviorName);
+                            if (props.onBehaviorsUpdated)
+                              props.onBehaviorsUpdated();
```

Comment:
Isn't it already handled by onRemoveBehavior method?

Reply:
yes good catch!